sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/VisualStudioCoverageXmlReportParserTest.java

```diff
@@ -95,12 +96,74 @@ public void valid_with_correct_file_language() throws Exception {
 
```

Comment:
I think it would be great to add a test also with multiple lines, some lines with only Get or Set covered and some with both.